### DISCUSSION TITLE
Backing up raw device

### DISCUSSION THREAD

**Discussion by oixdbb** [borgbase/vorta#1781] [FAQ]
Is it possible to backup a file in /dev/disk/by-uuid in Vorta? I'm running Vorta as root but devices don't show in /dev/.

**Comment by m3nu** [comment]
Could be due to the Qt file selector. It's set to files or folders. Anything in /dev may be a special file type and not show up.

We could quickly change that, though.

**Reply by jetchirag** [reply]
Couldn't reproduce this in Kali.

![image](https://github.com/borgbase/vorta/assets/15169678/4694c50e-aaf5-4b3e-8ca2-2b431f8ea1d3)

**Reply by real-yfprojects** [reply]
Also I couldn't find a attribute/method/property of QFileDialog to influence that.

**Comment by oixdbb** [comment]
That would be great
